### ISSUE TITLE
Error in normalizing layer

### ISSUE THREAD

**Issue by Ashbajawed** [GValiente/pocket-tensor#14]
Any solution for the following error

![image](https://user-images.githubusercontent.com/46400775/90362825-8924b800-e072-11ea-889e-f4ed862f8dca.png)

here are the basic checkpoint and model file
https://drive.google.com/file/d/1K6oDZa-wY7NAETi9fM2k_M7YYr6GWocd/view?usp=sharing

https://drive.google.com/file/d/1fuRGGQqsZEMvao7REpYcg95OONWJFd6U/view?usp=sharing

make_model.py (file generating this error ) consist of following code

from network_withoutSE import model
model= model((18,8,8), 1968 ,8)
model.load_weights("b1_episode4001_noSE.h5")

#save model
from pt import export_model
export_model(model, 'pocket_tensor.model')

**Comment by GValiente** [comment]
Hi,

Sorry, but this project is not in development anymore.

If you want to run trained Keras models from a C++ application, try [TensorFlow Lite](https://www.tensorflow.org/lite) and [frugally-deep](https://github.com/Dobiasd/frugally-deep).
